migrations/2023-05-02-102332_payout_create/down.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE PAYOUT_ATTEMPT;
+
+DROP TABLE PAYOUTS;
+
+DROP TYPE "PayoutStatus";
+
+DROP TYPE "PayoutType";
+
+-- Alterations
+
+ALTER TABLE
+    merchant_account DROP COLUMN payout_routing_algorithm;
+
+ALTER TABLE locker_mock_up DROP COLUMN enc_card_data;
+
+DELETE FROM pg_enum
+WHERE
+    enumlabel = 'payout_processor'
+    AND enumtypid = (
+        SELECT oid
+        FROM pg_type
+        WHERE typname = 'ConnectorType'
+    )
```

Comment:
We'd preferably want to avoid touching Postgres system/internal tables wherever possible.

Reply:
Sure